yt_dlp/extractor/arte.py

```diff
@@ -205,6 +228,7 @@ def _real_extract(self, url):
                 {'url': image['url'], 'id': image.get('caption')}
                 for image in metadata.get('images') or [] if url_or_none(image.get('url'))
             ],
+            'chapters': chapters,
```

Comment:
you were right to have this since not every video has chapters. then you can remove `chapters: []` from the tests. my bad
```suggestion
            'chapters': chapters or None,
```